client/src/ui/ModalView/ModalView.tsx

```diff
@@ -126,17 +137,31 @@ const ModalView = ({ isOpen, setIsOpen, card }: ModalViewProps) => {
         </ModalDescription>
       </ModalContent>
       <ModalActions>
-        <Button color="black" onClick={() => setIsOpen(false)}>
-          Add a review
-        </Button>
-        <Button
-          content="Worked with"
-          labelPosition="right"
-          icon="checkmark"
-          onClick={() => setIsOpen(false)}
-          positive
-        />
+        {user.role === "parents" && (
+          <>
+            <Button color="black" onClick={handleAddReview}>
+              Add a review
+            </Button>
+            <Button
```

Comment:
also here show a popup message saying thanks for updating. you can use import swal from 'sweetalert2'